### PR TITLE
add ability to log DEBUG messages to syslog from json option

### PR DESCRIPTION
**Pull request by pfrederiksen**
DEBUG messages now log to syslog if debug: 1

### CODE REVIEW

**Comment by asdil12** [comment]
You could simply run pymultimonaprs with the `--verbose` option.
Isn't that enough?

**Comment by pfrederiksen** [comment]
Sure, if you don't run it as a service. If you run it as a service without editing the init.d script (which is bad practice) you can use -v. If you want to run as a service and use the json config, where the option should be, you'll want it there.

One place for configuration seems reasonable.

**Comment by asdil12** [comment]
It does.
But the patches are very bad.

**Comment by pfrederiksen** [comment]
what would you rather have?

File: pymultimonaprs.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
 		"ppm": 0,
 		"gain": 0,
 		"offset_tuning": false,
-		"device_index": 0
+		"device_index": 0,
+		"debug": 0
```
Comment (asdil12) [inline pymultimonaprs.json]: - Wrong config section - this is not specific to the rtl backend
- Wrong value type - use bool true/false

Also I would call the key `verbose` to use the same name as for the cli argument

**Comment by pfrederiksen** [comment]
alright, that makes sense. I'll close this pull request and submit the changes you recommend. This is my first ever pull request, so be patient ;)

File: bin/pymultimonaprs [hunk 2]
```diff
@@ -23,7 +23,7 @@ args = parser.parse_args()
 config = json.load(open(args.config))
 
 logger = logging.getLogger('pymultimonaprs')
-loglevel = logging.DEBUG if args.verbose else logging.INFO
+loglevel = logging.DEBUG if (int(config['rtl']['debug'] == 1)) else logging.INFO
```
Comment (asdil12) [inline bin/pymultimonaprs]: You are creating a bool value, converting it to int and then let python autocast it back to bool.
Also you make pymultimonaprs ignore the --verbose cli argument but don't remove the cli-parser option. But I wouldn't remove that option - insted it should use debug mode if called with `--verbose` or if `verbose = true` in config.

Also you can't just expect that existing config files (installations that get this change via pkg update) have the `verbose` key. So you need to use the `dict.get(dict_key, failover_value)` function.

So the line would look like:
```python
loglevel = logging.DEBUG if args.verbose or config.get('verbose', False) else logging.INFO
```

**Pull request closed without merging**